src/Tracy/Debugger.php

```diff
@@ -470,6 +470,13 @@ public static function timer($name = NULL)
 		$now = microtime(TRUE);
 		$delta = isset($time[$name]) ? $now - $time[$name] : 0;
 		$time[$name] = $now;
+		if (!self::$productionMode) {
+			static $panel;
+			if (!$panel) {
+				$panel = self::getBar()->getPanel('time');
+			}
+			$panel->data[] = array('name' => $name, 'time' => $now, 'delta' => $delta);
+		}
```

Comment:
I agree with zero `$delta`. It makes sense to me. However usage of `$delta` only has no advantage against to `bd(Debugger::time())`. Maybe the default label is `bd(Debugger::time('foo'), 'foo')`. The information about timestamp of timer "event" and the time since last timer usage is the advantage.

The performance improvement you suggest is super micro optimization. I cannot measure any difference in visible `ms` units. It also affects development environment only so it does not matter in my opinion.